scripts/ci/lib.sh

```diff
@@ -1916,11 +1916,13 @@ junit_wrap() {
     local command_output_file
     command_output_file="$(mktemp)"
 
-    if "$@" 2>&1 | tee "${command_output_file}"; then
```

Comment:
Having `| tee` here creates a subshell which causes `export VAR=VALUE` not to propagate from wrapped commands. The flow of the upgrade test needs this propagation.